Gruntfile.js

```diff
@@ -85,8 +85,51 @@ module.exports = function(grunt) {
             }
         },
         clean: {
+            coverage: ['coverage/', 'coverage.lcov', '.nyc_output/'],
             dist: ['.dist/'],
+            packages: ['Consus-Client-*/'],
             test: ['.test/']
+        },
+        electron: {
+            win32Build: {
+                options: {
+                    name: 'Consus-Client',
+                    dir: '.',
+                    out: '.',
+                    version: '1.4.2',
+                    platform: 'win32',
+                    arch: 'x64',
+                    'no-prune': true,
+                    overwrite: true,
+                    asar: true,
+                    icon: 'consus-logo.ico' //This isn't working?
+                }
+            },
+            // macosBuild: {
```

Comment:
As long as you can package it properly I'm fine with using it in the final version of this branch.  I just made this PR now because that's what you and Trevin seemed to want.